.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   # Common versions
   GO_VERSION: '1.17'
-  GOLANGCI_VERSION: 'v1.42'
+  GOLANGCI_VERSION: 'v1.45.2'
```

Comment:
I updated to the latest version and skip cache in #43 , that would fix the ci issue.